pkg/server/list_users.go

```diff
@@ -110,6 +110,7 @@ func (s *Server) ListUsers(
 		methodName,
 		utils.Bucketize(uint(datastoreQueryCount), s.requestDurationByQueryHistogramBuckets),
 		utils.Bucketize(uint(dispatchCount), s.requestDurationByDispatchCountHistogramBuckets),
+		"N/A",
```

Comment:
I don't like this, but we can't reuse the same histogram without doing this as if we don't pass the expected number of label values then we will panic. I did some digging and there doesn't seem to be any alternative other than creating a separate histogram

Reply:
Scratch that, it just never got added to the ListUsersRequest. I'll add it in this PR!